compiler/rustc_passes/src/stability.rs

```diff
@@ -886,6 +922,17 @@ impl<'tcx> Visitor<'tcx> for CheckTraitImplStable<'tcx> {
     }
 }
 
+pub fn check_const_impl_trait(tcx: TyCtxt<'_>) {
+    let features = tcx.features(); // FIXME How cheap is this call?
```

Comment:
How cheap is `tcx.features()`? Should I just inline this variable?

Reply:
Should be increadibly cheap, but if you are using the output twice, this is better.

This function may go away anyway if it's possible to merge the logic into the general stability checking.